### PR TITLE
Milestone 9 : Top mobile navigation menu

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Responsive top navbar : Milestone 9
In this Milestone I was able to:
- Learn and practice good Javascript practices
- Learn and practice basic DOM manipulation
- Practice how to use Javascript linter
- Practice how to document my projects professionally

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,9 +72,9 @@ <h2><a href="#">Nemwel Boniface</a></h2>
 
         <div class="navLinks">
           <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#aboutme">About</a></li>
+            <li><a href="#contact">Contact</a></li>
```
Comment (leolpaz) [inline index.html:77]: - [x] Great job with the menu, but as per the requirement screenshotted below the menu should disappear, while it changes to the correct part of the page, the menu is still up there over the "authordetails" section, kindly make it in a way that the menu closes with similar behavior to pressing the x button
image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Nemwel-Boniface , 

You've done an amazing job so far, I'd like to highlight the following:
- Menu looks very similar to Figma design
- Options on menu take you to the correct part of the page

Some issues need to be resolved before your approval, but you're almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nemwel-Boniface) [reply index.html:77]: Hello @leolpaz thank you for this review. I have managed to make the menu bar disappear when you click the link. I have used a for..of loop and I am getting this error when running my linters. Any ideas of help please ? 
"iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations"

Comment (leolpaz) [reply index.html:77]: Can you try using forEach on lists instead of the type of loop you're using?

Comment (Nemwel-Boniface) [reply index.html:77]: Hello @leolpaz  I tried using forEach and it worked! thank you

**Review by rustam-tolipov — APPROVED** [review]
## Hi TEAM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**